addons/l10n_account_edi_ubl_cii_tests/tests/common.py

```diff
@@ -225,7 +225,7 @@ def _generate_move(self, seller, buyer, send=True, **invoice_kwargs):
             account_move._generate_pdf_and_send_invoice(self.move_template)
         return account_move
 
-    def _assert_invoice_attachment(self, attachment, xpaths, expected_file):
+    def _assert_invoice_attachment(self, attachment, xpaths, expected_file_path):
```

Comment:
either cherry-pick Brice's commit or make a separate commit for this changes, but don't squash it with your other changes, it has nothing to do with the Romanian EDI :)